### PR TITLE
Kick players by name

### PR DESCRIPTION
**Pull request by MarbleTurtle**
Changes the "Attempting to kick player from friends chat..." message to include the name instead of player.

### CODE REVIEW

File: runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java [hunk 1]
```diff
@@ -150,11 +150,22 @@ default boolean confirmKicks()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "kickWithName",
+			name = "Kick with name",
```
Comment (raiyni) [inline runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java:155]:
```suggestion
			name = "Show kicked player",
```

Or something with more intent

Comment (MarbleTurtle) [reply runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java:155]: Kick player by name? More obvious what it does at a glance

Comment (raiyni) [reply runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java:155]: This is just adding the kicked players name to the game message. It should reflect that. Your sentence implies you are turning on an option to type a players name in to kick them.

File: runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java [hunk 2]
```diff
@@ -150,11 +150,22 @@ default boolean confirmKicks()
 		return false;
 	}
 
+	@ConfigItem(
+			keyName = "kickWithName",
```
Comment (Adam-) [inline runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatConfig.java:154]: Fix the formatting here.

**Pull request merged**